book/09-git-and-other-scms/sections/import-svn.asc

```diff
@@ -67,63 +67,63 @@ Date:   Sun May 3 00:12:22 2009 +0000
-First you'll move the tags so they're actual tags rather than strange remote branches, and then you'll move the rest of the branches so they're local.
+Kailangan mo ring gumawa ng kakaunti paglilinis sa post-import. 
+Para sa isang bagay, dapat mong linisin ang kakaibang mga reperensya na itinakda ng `git svn`.
+Una ililipat mo ang mga tag upang sila ay maging aktwal na mga tag sa halip na kakaibang remote na mga branch, at pagkatapos ay ililipat mo ang natitirang mga branch upang sila ay maging lokal.
 
-To move the tags to be proper Git tags, run:
+Upang maglipat ng mga tag upang maging nararapat na Git na mga tag, patakbuhin ang:
 
 [source,console]
 ----
 $ for t in $(git for-each-ref --format='%(refname:short)' refs/remotes/tags); do git tag ${t/tags\//} $t && git branch -D -r $t; done
 ----
 
-This takes the references that were remote branches that started with `refs/remotes/tags/` and makes them real (lightweight) tags.
+Kukunin nito ang mga reperensya na remote na mga branch na nagsisimula sa `refs/remotes/tags/` at gagawin silang tunay (magaan) na mga tag.
 
-Next, move the rest of the references under `refs/remotes` to be local branches:
+Susunod, ilipat ang natitira sa mga reperensya sa ilalim ng `refs/remotes` upang maging lokal ng mga branch:
 
 [source,console]
 ----
 $ for b in $(git for-each-ref --format='%(refname:short)' refs/remotes); do git branch $b refs/remotes/$b && git branch -D -r $b; done
 ----
 
-It may happen that you'll see some extra branches which are suffixed by `@xxx` (where xxx is a number), while in Subversion you only see one branch.
-This is actually a Subversion feature called ``peg-revisions'', which is something that Git simply has no syntactical counterpart for.
-Hence, `git svn` simply adds the svn version number to the branch name just in the same way as you would have written it in svn to address the peg-revision of that branch.
-If you do not care anymore about the peg-revisions, simply remove them:
+Maaaring mangyari na makikita mo ang ilang karagdagang mga branch na naka-suffix ng `@xxx` (kung saan ang xxx ay isang numero), habang sa Subversion makakakita ka lamang ng isang branch.
+Ito ay isang aktwal na tampok ng Subversion na tinatawag na ``peg-revisions'', na isang bagay na simpleng walang sintaktikal na katumbas sa Git.
+Kaya, ang `git svn` ay simpleng nagdaragdag ng numero ng svn na bersyon sa pangalan ng branch sa parehong paraan lamang katulad ng pagsulat mo nito sa svn upang isalaysay ang peg-revision sa branch na iyon.
+Kung wala ka nang pakialam tungkol sa mga peg-revision, simpleng burahin ang mga iyon:
 
 [source,console]
 ----
 $ for p in $(git for-each-ref --format='%(refname:short)' | grep @); do git branch -D $p; done
 ----
 
-Now all the old branches are real Git branches and all the old tags are real Git tags.
+Ngayon lahat ng matandang mga branch ay tunay na Git na mga branch at lahat ng matandang mga tag ay tunay na Git na mga tag.
```

Comment:
`matandang mga branch` - could be `mga lumang branch`.
the same goes with `matandang mga tag`

Reply:
haha this made me laugh sorry. changing it now